docs/index.html

```diff
@@ -45,6 +55,7 @@
     </script>
     <script type="text/javascript">
       const options = {
+        highlightLines: true
```

Comment:
The contents in this file are identical to those in #31, but it hasn't been approved and merged yet.